azure-kusto-data/source/connectionBuilder.js

```diff
@@ -58,7 +58,7 @@ module.exports = class KustoConnectionStringBuilder {
         if (!connectionString || connectionString.trim().length === 0) throw new Error("Missing connection string");
 
         if (!!connectionString && connectionString.split(";")[0].indexOf("=") === -1) {
-            connectionString = "Data Source=" + connectionString;
+            connectionString = "Data Source=" + connectionString.replace(/\/$/, "");
```

Comment:
Why do we need that? Eventually we create URL from that, and it is valid whether we have / at the end or not.

Reply:
https://github.com/Azure/azure-kusto-node/blob/95a3ad17a952fb780cf73b7e2385f4a6c723d933/azure-kusto-data/source/client.js#L17

Otherwise we will get something like `https://cluster.region.kusto.windows.net//v2/rest/query/`.

The other option is when adding the postfix, we can check if a slash is needed

Reply:
So can we do something like you did at:
https://github.com/Azure/azure-kusto-node/blob/95a3ad17a952fb780cf73b7e2385f4a6c723d933/azure-kusto-data/source/security.js#L17

Just for clarity.